examples_tests/18.MitsubaLoader/main.cpp

```diff
@@ -418,6 +436,8 @@ int main()
 					modifiedShaders.insert({ core::smart_refctd_ptr<asset::ICPUSpecializedShader>(fs),newfs });
 					pipeline->setShaderAtStage(asset::ICPUSpecializedShader::ESS_FRAGMENT, newfs.get());
 				}
+				// invert what is recognized as frontface in case of RH camera
+				pipeline->getRasterizationParams().frontFaceIsCCW = !leftHandedCamera;
```

Comment:
you'll need to check whether to invert the front-back face boolean, per-instance anyway because of world tform

Reply:
i know, its done

Reply:
I mean that you're not going to have fun if you have multiple cameras with different handedness and you need to change the non dynamic pipeline parameter of triangle winding order for each of them.

So instead of handling it here, pass it to the shader though a uniform or a PC and combine with world matrix determinant.